cloud-aws-native/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNativeResourceConnector.java

```diff
@@ -95,6 +96,11 @@ public String getDBStackTemplate() throws TemplatingNotSupportedException {
         throw new TemplatingNotSupportedException();
     }
 
+    @Override
+    public List<AwsInstanceStorageInfo> getInstanceTypeEphemeralInfo(AuthenticatedContext authenticatedContext, List<String> instanceTypes) {
+        return null;
```

Comment:
please return an empty list instead of a null. the latter is more prone to error since the return type is not even marked as nullable

Reply:
removed this method